zzz_generated/app-common/app-common/zap-generated/attributes/Accessors.cpp

```diff
@@ -972,6 +972,36 @@ EmberAfStatus Set(chip::EndpointId endpoint, int16_t value)
 
 } // namespace HighTempThreshold
 
+namespace LowTempDwellTripPoint {
+
+EmberAfStatus Get(chip::EndpointId endpoint, uint32_t * value)
+{
+    return emberAfReadServerAttribute(endpoint, Clusters::DeviceTemperatureConfiguration::Id, Id, (uint8_t *) value,
+                                      sizeof(*value));
+}
+EmberAfStatus Set(chip::EndpointId endpoint, uint32_t value)
+{
+    return emberAfWriteServerAttribute(endpoint, Clusters::DeviceTemperatureConfiguration::Id, Id, (uint8_t *) &value,
+                                       ZCL_INT24U_ATTRIBUTE_TYPE);
```

Comment:
And this is writing native-endian, and the first 3 bytes, which might be the three high bytes or the three low bytes, depending on endianness.